src/agent/srcview/src/pdbcache.rs

```diff
@@ -49,7 +49,7 @@ impl PdbCache {
 
             while let Some(symbol) = symbols.next()? {
                 if let Ok(SymbolData::Procedure(proc)) = symbol.parse() {
-                    let mut lines = program.lines_at_offset(proc.offset);
+                    let mut lines = program.lines_for_symbol(proc.offset);
```

Comment:
I dug around a bit, and this appears to be slightly more than an NFC rename. However, the MASM-specific edge case being addressed (bounds discrepancies between nominal symbol offset ranges and line info offsets) shouldn't impact this code.

It _would_ be relevant if we e.g. round-tripped from an offset to a `LineInfo` and back to an offset, and then indexed directly from the offset _into something_ like a borrowed slice of a raw PDB file. However, the offsets here are `PdbInternalSectionOffset` values, which we only ever pass back to `pdb` crate APIs, so I think we're fine.